terra2/assetlist.json

```diff
@@ -69,6 +69,28 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/ampluna.svg"
         },
         "coingecko_id": "eris-amplified-luna"
+      },
+      {
+        "description": "Lion DAO is a community DAO that lives on the Terra blockchain with the mission to reactivate the LUNAtic community and showcase Terra protocols & tooling",
+        "type_asset": "cw20",
+        "address": "terra1lxx40s29qvkrcj8fsa3yzyehy7w50umdvvnls2r830rys6lu2zns63eelv",
+        "denom_units": [
+          {
+            "denom": "cw20:terra1lxx40s29qvkrcj8fsa3yzyehy7w50umdvvnls2r830rys6lu2zns63eelv",
+            "exponent": 0
+          },
+          {
+            "denom": "roar",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:terra1lxx40s29qvkrcj8fsa3yzyehy7w50umdvvnls2r830rys6lu2zns63eelv",
+        "name": "Lion DAO",
+        "display": "roar",
+        "symbol": "ROAR",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/roar.png"
```

Comment:
you have a png file in the svg property. You can just rename the property to "png"